data-raw/fontawesome-update.R

```diff
@@ -166,6 +165,14 @@ for (i in seq_along(fa_list)) {
   fa_tbl <- as.data.frame(fa_tbl)
 }
 
+# Use `iconv()` to perform transliteration to ASCII and also remove all
+# generated/extant single and double quotes
+fa_tbl <-
+  fa_tbl %>%
+  dplyr::mutate(label = iconv(label, "UTF-8", "ASCII//TRANSLIT")) %>%
```

Comment:
I think this should use `stringi::stri_escape_unicode()`, since this `iconv()` call will change the content. Then you also don't need to remove quotes.

```R
iconv("é", "UTF-8", "ASCII//TRANSLIT")
#> [1] "'e"
stringi::stri_escape_unicode("é")
#> [1] "\\u00e9"
```